vignettes/geological-completeness.Rmd

```diff
@@ -363,6 +363,75 @@ ggplot(counts, aes(fill = environment, y = count, x = mid_ma)) +
 
 And that's it! Not too bad, right? There are some clear differences between our generated plots and those from the original [Peters & Heim (2010)](https://www.jstor.org/stable/25609444) paper (Fig. 14), which is to be expected given that substantially more data is now available in [Macrostrat](https://macrostrat.org) (e.g., no "unknown" paleoenvironments), and there may be differences in what we defined as "marginal" environments. Despite these differences, some broad-scale patterns are largely consistent, such as the higher proportion of terrestrial lithostratigraphic units in the Mesozoic and Cenozoic, compared to the Paleozoic.
 
+# Bonus plot: rock types
```

Comment:
```suggestion
# Bonus plot: rock classes
```
I only make this suggestion because the request is `lithology_class = `... but I don't think it matters much!